pramen/core/src/main/scala/za/co/absa/pramen/core/runner/orchestrator/DependencyResolverImpl.scala

```diff
@@ -62,7 +62,7 @@ class DependencyResolverImpl(deps: Seq[JobDependency]) extends DependencyResolve
   }
 
   override def getDag(outputTables: Seq[String]): String = {
-    val tables = if (outputTables.isEmpty) {
+    val tables = if (outputTables.nonEmpty) {
```

Comment:
I think maybe those were accidentally switched, right? Because I think right now this branch would not have any effect in the empty Seq

Reply:
Yes, looks like it happened exactly like that